v2.1/online-schema-changes.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Online Schema Changes
+summary: Update table schemas without external tools or downtime.
+toc: true
+---
+
+CockroachDB's online schema changes provide a simple way to update a table schema without imposing any negative consequences on an application — including downtime. The schema change engine is a built-in feature requiring no additional tools, resources, or ad hoc sequencing of operations.
+
+Benefits of online schema changes include:
+
+- Changes to your table schema happen while the database is running.
+- The schema change runs as a [background job][show-jobs] without holding locks on the underlying table data.
+- Your application's queries can run normally, with no effect on read/write latency. The schema is cached for performance.
+- Your data is kept in a safe, [consistent][consistent] state throughout the entire schema change process.
+
+{{site.data.alerts.callout_success}}
```

Comment:
This tip and the note following feel somehow redundant. Is there are way to combine them? I think the second callout is more important here, if you need to choose. Perhaps you can put the recommendation about only doing one schema change per transaction in the Limitations section...

Reply:
Fixed by moving the second callout into the *Limitations* section and consolidating some of the language there.